hack/make-rules/test-cmd-util.sh

```diff
@@ -1613,7 +1613,7 @@ run_secrets_test() {
 
   ### Create a docker-registry secret in a specific namespace
   # Pre-condition: no SECRET exists
-  kube::test::get_object_assert 'secrets --namespace=test-secrets' "{{range.items}}{{$id_field}}:{{end}}" ''
+  kube::test::wait_object_assert 'secrets --namespace=test-secrets' "{{range.items}}{{$id_field}}:{{end}}" ''
```

Comment:
There was some discussion in https://github.com/kubernetes/kubernetes/pull/33414.
The idea was that since both servers have the same API, we should be able to reuse the test suite.